src/os2datascanner/projects/admin/adminapp/templates/miniscan.html

```diff
@@ -56,6 +62,27 @@ <h2>{% trans "File to scan" %}</h2>
               {% trans "Please, select another (smaller) file to scan." %}
             </div>
           {% endwith %}
+
+          <h2>{% trans "Enter your text below" %}</h2>
+          <input name="text"
+                 id="upload-text"
+                 type="text"
+<<<<<<< HEAD
+                 class="upload" />
+=======
+                 class="upload"
+                 spellcheck="true"
+                 placeholder="..."
+                 maxlength="2000"> 
+          </textarea>
+          <br><br>
+          <input name="KLE-switch"
+                  id="kle-switch"
+                  type="checkbox"
+                  class="upload">{% trans "Enable classification (slower)" %}
+          </input>
+>>>>>>> 610b29719 (Added placeholder='...' to avoid wierd spaces in textfield when reloading page)
```

Comment:
This is Git conflict resolution stuff that really shouldn't be in this branch (or any branch!).

Reply:
Fixed !